modules/mri_violations/test/TestPlan.md

```diff
@@ -1,47 +1,64 @@
+     are set in the filters of that page)
+     - the 'MRI Protocol Violations' page listing all protocol violations 
+     with the MRI protocol parameters showing as a first table (check that 
+     correct scan information are set in the filters of that page) 
     [Manual Testing]
-4.  MRI Protocol Violations page check Selection Filter results for each of DCCID, PSCID, PatientName, SeriesUID,
-    Time Run, Series Description
+4.  MRI Protocol Violations page check Selection Filter results for each of 
+    DCCID, PSCID, PatientName, SeriesUID, Time Run, Series Description
     [Automation Testing]
-5.  MRI Protocol Violations page - ensure first table displays all valid protocols
-    for the study
+5.  MRI Protocol Violations page - ensure first table displays all valid 
+    protocols for the study
     [Manual Testing]
 6.  MRI Protocol Violations page - displays header information of violated scan
     [Manual Testing]
-7.  MRI Protocol Violations page - allows editing of protocol table with permission 'Violated Scans: Edit MRI protocol table'.
+7.  MRI Protocol Violations page - allows editing of protocol table with 
+    permission 'Violated Scans: Edit MRI protocol table'.
     Edit values in the protocol table and verify it is saved in the database.
     [Manual Testing]
-8.  Verify one or two of the scans in the second table and ensure it indeed violates mri protocol.
+8.  Verify one or two of the scans in the second table and ensure it indeed 
+    violates the MRI protocol.
     [Manual Testing]
-9.  Filter for scans that violate MRI check  (Type of Problem : MRI Protocol Check violation). Ensure link
-    under Problem column goes to MRI Protocol Check Violations page with correct scan information
+9.  Filter for scans that violate MRI checks  ('Type of Problem' = 'Protocol 
+    Violation'). Ensure link under the 'Problem' column goes to the MRI 
+    Protocol Check Violations page with correct scan information
     [Manual Testing]
-10. MRI Protocol Check violation page check Selection Filter results for each of Tarchive ID, PatientName, CandID
-    DICOM Series UID
+10. MRI Protocol Check Violations page - check Selection Filter results for 
+    each of Tarchive ID, PatientName, CandID and DICOM Series UID
     [Manual Testing]
-11. MRI Protocol Check violation page - click on link under PatientName column, should go to Dicom Archive for the
-    patient if permission 'Across all sites view Dicom Archive modile and pages' is granted
+11. MRI Protocol Check Violations page - click on link under PatientName 
+    column, should go to DICOM Archive for the patient if permission 'Across 
+    all sites view DICOM Archive module and pages' is granted
     [Manual Testing]
 12. Upload scan that violates protocol and ensure it shows up in the module
     [Manual Testing]
-13. Upload scan that violates protocol check and ensure it shows up in the module
+13. Upload scan that violates protocol checks and ensure it shows up in the 
```

Comment:
12 and 13 are not the same thing?

Reply:
woops sorry, this review was supposed to have my other comments too

Reply:
actually, they are different. I tried rephrasing it. Hopefully it makes more sense. Those two are so confusing...